src/api2/managers/documents-manager.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios'
+import FormData from 'form-data'
+
+import _kebabCase from 'lodash/kebabCase'
+import _omit from 'lodash/omit'
+
+import { ApiCaller } from '../api-caller'
+
+const HEADER_CONTENT_TYPE = 'Content-Type'
+const MIME_TYPE_MULTIPART_FORM_DATA = 'multipart/form-data'
+
+/**
+ * DocumentsManager uploads a document to the storage server
+ */
+export class DocumentsManager {
```

Comment:
consider adding a method to get a file
or to build the url to get it

Reply:
Added